kore/src/Kore/Step/Simplification/Exists.hs

```diff
@@ -319,10 +312,11 @@ quantifyPattern variable original@Conditional { term, predicate, substitution }
     , "variable=" ++ unparseToString variable
     , "patt=" ++ unparseToString original
     ]
-  | quantifyTerm = mkExists variable <$> original
+  | quantifyTerm = markSimplified . mkExists variable <$> original
   | quantifyPredicate =
     Conditional.withCondition term
-    $ Predicate.fromPredicate
+    -- TODO (thomas.tuegel): This could probably be simplified further.
```

Comment:
Could you give some hints about what kind of simplification you had in mind?

Reply:
Yes, I only meant that we are constructing an `\and` without invoking the corresponding simplifier. I will make the comment more specific.